### DISCUSSION TITLE
Evaluation of usage behavior - KPI´s

### DISCUSSION THREAD

**Discussion by VisFlM2024** [community/community#123012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello all,

short example:
A team has 50 CoPilot licenses in use.

Question:
Which options do we have to monitor usage and benefits?
It is only a matter of recognizing and documenting the benefits.

Are there dashboards for KPIs in the Management Console?

Thank you for your help.

**Comment by Jaaziel-Polanco** [comment]
Hello,

To monitor the usage and benefits of GitHub Copilot licenses, you can consider the following options:

Usage Monitoring:
Organization admins can view the usage for GitHub Copilot in their organization. [If you have a subscription to GitHub Copilot Business or GitHub Copilot Enterprise, you can view usage information for GitHub Copilot in your organization, broken down by seat assignment status](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage)[1](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage).
To view your organization’s use of GitHub Copilot, go to the upper-right corner of GitHub.com, select your profile photo, then click “Your organizations”. Next to the organization, click “Settings”. In the “Access” section of the sidebar, click “Billing and plans”. [Under “Copilot”, you can view the breakdown of your GitHub Copilot usage and upcoming changes in your organization](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage)[1](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage).
Benefits Documentation:
[GitHub Copilot benefits developers and enterprises by helping developers solve common programming problems up to 55% faster, allowing them to focus on higher-value work to deliver better products and services](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage)[2](https://resources.github.com/learn/pathways/copilot/essentials/what-github-copilot-can-do-for-your-organization/).
[It also keeps your data secure on trusted Microsoft Azure infrastructure and screens code suggestions through multiple filters, such as a toxicity filter and code quality filter, as well as optional code referencing filters](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage)[2](https://resources.github.com/learn/pathways/copilot/essentials/what-github-copilot-can-do-for-your-organization/).
KPI Dashboards:
While there is no direct mention of specific dashboards for KPIs in the Management Console related to GitHub Copilot, you can utilize tools like the GitHub Copilot Metrics Viewer, which is currently in public beta. [This tool visualizes various metrics provided via the Copilot Business Metrics API, such as acceptance rate, total suggestions, total acceptances, total lines suggested, total lines accepted, total active users, and languages breakdown analysis](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-copilot/viewing-your-github-copilot-usage)[3](https://github.com/github-copilot-resources/copilot-metrics-viewer).
Additionally, you can use the Copilot Usage Dashboard, an Angular application designed to provide insights into GitHub Copilot usage within an organization. [It utilizes the Copilot Usage Metrics API (private Beta) and Copilot Seat Management API to fetch and display relevant data](https://github.com/octodemo/Copilot-Usage-Dashboard)[4](https://github.com/octodemo/Copilot-Usage-Dashboard).
